src/deck-options.md

```diff
@@ -5,12 +5,12 @@
 Deck options primarily control the way Anki schedules cards. It is recommended
 that you spend a few weeks with the defaults to get a feel for how Anki works
 before you start adjusting the options. Please make sure you understand the options
-before changing them, as mistakes could reduce Anki's effectiveness.
+before changing them, as mistakes can reduce Anki's effectiveness.
 
-If you're on the desktop app, do any of the following to access deck options:
+If you're on the desktop app, do any of the following to open deck options:
```

Comment:
I think the logic behind this was that the tooltips direct people to this manual and so I didnt want to make assumptions about the reader.

Reply:
It's unfortunate if the tooltips in the other apps aren't linking to their own manuals, but if we're going to try to serve all of the audiences, we should include all of the instructions -- or at least link out to the Droid and Mobile instructions.

Reply:
To not make it confusing it should mention "On desktop" or something similar. But linking to other manuals wouldn't be necessary. If someone is accessing it from mobile, they already know how to access it.

The language here was bad, I should've said "On desktop" instead of "If you're on desktop".

Reply:
It looks like you've now changed it to: 
> On your computer, do any of the following to open deck options

I'll defer to Damien - https://github.com/ankitects/anki-manual/pull/250#discussion_r1752323650